src/multipleCenters.js

```diff
@@ -0,0 +1,68 @@
+// array of all centers
+const centers = []
```

Comment:
I think the approach of storing centers in a list is a fine idea for this exercise; later on, you will have a few more tools that will allow you to model this extra requirement a lot better!